Sample/SextantSample.UWP/Views/RedView.xaml.cs

```diff
@@ -0,0 +1,51 @@
+﻿using System;
+using ReactiveUI;
+using SextantSample.Core;
+using SextantSample.Core.ViewModels;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+
+// The Blank Page item template is documented at https://go.microsoft.com/fwlink/?LinkId=234238
+
+namespace SextantSample.UWP.Views
+{
+    /// <summary>
+    /// An empty page that can be used on its own or navigated to within a Frame.
+    /// </summary>
+    public sealed partial class RedView : Page, IViewFor<RedViewModel>
+    {
+        public RedView()
+        {
+            this.InitializeComponent();
+
+            this.WhenActivated(d =>
```

Comment:
Need `WhenActivated`?

Reply:
Honestly, I don't know if `WhenActivated` is required here.  I know UWP are easy to leak memory from view objects that reference external things... like here?  I'll take it out just because it won't matter for a sample app.